languages/go/internal/host/host.go

```diff
@@ -331,7 +331,16 @@ func (h Host) ToPolar(v interface{}) (*Value, error) {
 		if err != nil {
 			return nil, err
 		}
-		repr := fmt.Sprintf("%v", v)
+		var repr string
+		value := reflect.ValueOf(v)
+		if value.Kind() == reflect.Ptr || value.Kind() == reflect.Interface {
```

Comment:
Isn't this the same thing we have on line 292? Are we missing the interface check there or something?